src/components/login.js

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {Component, PropTypes} from 'react';
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {Image, StyleSheet, Text, TextInput, View} from 'react-native';
+import {Actions as Routes} from 'react-native-router-flux';
+
+import * as loginActions from 'actions/login';
+import Button from 'components/button';
+import ErrorText from 'components/error_text';
+import {GlobalStyles} from 'styles';
+import logo from 'images/logo.png';
+
+const styles = StyleSheet.create({
+    container: {
+        flex: 1,
+        flexDirection: 'column',
+        alignItems: 'center',
+        paddingTop: 200,
+        backgroundColor: 'white'
+    },
+    logo: {
+        marginBottom: 10
+    }
+});
+
+const propTypes = {
+    login: PropTypes.object.isRequired,
+    actions: PropTypes.object.isRequired
+};
+
+class Login extends Component {
+    static propTypes = propTypes;
+    state = {
+        loginId: '',
+        password: ''
+    };
+
+    componentWillReceiveProps(props) {
+        if (this.props.login.status === 'fetching' &&
+          props.login.status === 'fetched') {
+            Routes.SelectTeam(); // eslint-disable-line new-cap
```

Comment:
@hmhealey commented on this in #22.  I changed all the route keys to goTo<component>.  We discussed this offline for a min but we should make the call here or in #22.